src/content/administration/dashboard.mdx

```diff
@@ -70,3 +70,42 @@ For each namespace, an administrator will be able to:
 The Secrets section of the Admin dashboard lets you define cluster-wide secrets. Using the `$SECRET_NAME` syntax, you can refer to these secrets directly in your Okteto manifest. They are also injected into the [environment running](/docs/cloud/okteto-cli/#built-in-tools-when-deploying-to-okteto-cloud) the commands in the `deploy` section of the manifest.
 
 > Secrets [defined by the user](/docs/cloud/secrets/) take precedence over the secrets defined here in case of a name collision.
+
+To inject these secrets into your Dockerfile, we recomment to use build secrets.
+
+### Build Secrets Sample
+
+Build secrets is the recommended way to inject secrets into your Dockerfile build process.
+
+For example, to mount your `.npmrc` file into your Dockerfile, you can have the following okteto manifest:
+
+```yaml
+build:
+  api:
+    context: .
+    secrets:
+      npmrc: .npmrc
+```
+
+The following Dockerfile:
+
+```
+FROM node:14
+
+WORKDIR /usr/src/app
+COPY package.json yarn.lock .
+
+RUN --mount=type=secret,id=npmrc,dst=/root/.npmrc yarn install
+```
+
+And the content of the `.npmrc` file would be:
+
+```
+registry=https://registry.npmjs.org/
```

Comment:
This is too much indirection for a sample. I propose we do the following instead?
1. A simple sample on how to use build secrets to pass an API key or a secret string
2. This specific sample as a how to in community.okteto.com

Reply:
Yes, I was thinking about that. A quick description in the build secrets docs, and this sample in the community